assets/components/organisms/footer/footer-legal.twig

```diff
@@ -15,7 +15,7 @@
 <div class="footer-legal">
   <div class="footer-legal-links">
     <a href="https://epfl-idevelop.github.io/elements/#/doc/accessibility.html">Accessibility</a>
-    <a href="https://mediacom.epfl.ch/disclaimer">Legals</a>
+    <a href="https://www.epfl.ch/about/overview/overview/regulations-and-guidelines">Legals</a>
```

Comment:
Le texte du lien : "Legals"
Le lien: "regulations-and-guidelines"
Le titre de la section dans la page: "Disclaimer"

On lance un dé et on choisit ?